cmd/primarycmd/describe.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package primarycmd
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"os"
+
+	"github.com/ava-labs/avalanche-cli/pkg/evm"
+	"github.com/ava-labs/avalanche-cli/pkg/key"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/networkoptions"
+	"github.com/ava-labs/avalanche-cli/pkg/subnet"
+	"github.com/ava-labs/avalanche-cli/pkg/utils"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/avalanchego/utils/units"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+)
+
+const art = `
+   _____       _____ _           _         _____
+  / ____|     / ____| |         (_)       |  __ \
+ | |   ______| |    | |__   __ _ _ _ __   | |__) |_ _ _ __ __ _ _ __ ___  ___ 
+ | |  |______| |    | '_ \ / _  | | '_ \  |  ___/ _  | '__/ _  | '_   _ \/ __|
+ | |____     | |____| | | | (_| | | | | | | |  | (_| | | | (_| | | | | | \__ \
+  \_____|     \_____|_| |_|\__,_|_|_| |_| |_|   \__,_|_|  \__,_|_| |_| |_|___/
+`
+
+var describeSupportedNetworkOptions = []networkoptions.NetworkOption{networkoptions.Local, networkoptions.Cluster}
+
+// avalanche primary describe
+func newDescribeCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "describe",
+		Short:        "Print details of the primary network configuration",
+		Long:         `The subnet describe command prints details of the primary network configuration to the console.`,
+		SilenceUsage: true,
+		RunE:         describe,
+		Args:         cobra.ExactArgs(0),
+	}
+	networkoptions.AddNetworkFlagsToCmd(cmd, &globalNetworkFlags, false, describeSupportedNetworkOptions)
+	return cmd
+}
+
+func describe(_ *cobra.Command, _ []string) error {
+	network, err := networkoptions.GetNetworkFromCmdLineFlags(
+		app,
+		globalNetworkFlags,
+		false,
+		describeSupportedNetworkOptions,
+		"",
+	)
+	if err != nil {
+		return err
+	}
+	fmt.Println(logging.LightBlue.Wrap(art))
+	var (
+		teleporterMessengerAddress string
+		teleporterRegistryAddress  string
+	)
+	if network.Kind == models.Local {
+		extraLocalNetworkData, err := subnet.GetExtraLocalNetworkData(app)
```

Comment:
nit: can be shortened with if extraLocalNetworkData, err := subnet.GetExtraLocalNetworkData(app); err!=nil... else 
		teleporterMessengerAddress = extraLocalNetworkData.CChainTeleporterMessengerAddress
		teleporterRegistryAddress = extraLocalNetworkData.CChainTeleporterRegistryAddress